moveit2_tutorials.repos

```diff
@@ -23,3 +23,7 @@ repositories:
     type: git
     url: https://github.com/PickNikRobotics/rviz_visual_tools.git
     version: 4.1.2
+  launch_param_builder:
```

Comment:
Do we need this? it''s already release for galactic https://github.com/ros/rosdistro/blob/master/galactic/distribution.yaml